transports/rabbitmq/index.md

```diff
@@ -14,6 +14,8 @@ tags:
 
 Provides support for sending messages over [RabbitMQ](http://www.rabbitmq.com/) using the [RabbitMQ .NET Client](https://www.nuget.org/packages/RabbitMQ.Client/).
 
+WARNING: The transport is not compatible with versions of the RabbitMQ broker earlier than 3.4.0.
```

Comment:
@SimonCropp I wanted to edit this, but I'm struggling to come up with the right language based on those guidelines. Would it be necessary to flip the direction of the constraint? E.g.

> The transport is only compatible with Versions 3.4.0 and above of the RabbitMQ broker.